test/tools/sanitize_json.d

```diff
@@ -13,6 +13,24 @@ import std.stdio;
 bool keepDeco = false;
 enum rootDir = __FILE_FULL_PATH__.dirName.dirName;
 
+// JSONType has been introduced in 2.082
+static if (__VERSION__ <= 2081) {
+    alias JSONType = JSON_TYPE;
+    alias JSON_TYPE_NULL = JSON_TYPE.NULL;
+    alias JSON_TYPE_OBJECT = JSON_TYPE.OBJECT;
+    alias JSON_TYPE_STRING = JSON_TYPE.STRING;
+    alias JSON_TYPE_ARRAY = JSON_TYPE.ARRAY;
+    alias JSON_TYPE_INTEGER = JSON_TYPE.INTEGER;
+    alias JSON_TYPE_UINTEGER = JSON_TYPE.UINTEGER;
+} else {
+    alias JSON_TYPE_NULL = JSONType.null_;
+    alias JSON_TYPE_OBJECT = JSONType.object;
+    alias JSON_TYPE_STRING = JSONType.string;
+    alias JSON_TYPE_ARRAY = JSONType.array;
+    alias JSON_TYPE_INTEGER = JSONType.integer;
+    alias JSON_TYPE_UINTEGER = JSONType.uinteger;
+}
+
```

Comment:
Why not the other way around ?
```D
// JSONType has been introduced in 2.082
static if (__VERSION__ <= 2081) {
    enum JSONType : JSON_TYPE
    {
        null_ = JSON_TYPE.NULL;
        // ...
    }
}
```

This way you only have an import in the `else` and when we update the base version it's only removing code.

Reply:
Comparison of different enum types (std.json defines its `JsonType` enum).

Reply:
Erg that's really stupid. But again, `enum` are really stupid.